### ISSUE TITLE
Close any desktop from Desktop View

### ISSUE THREAD

**Issue by benz145** [mzomparelli/zVirtualDesktop#203]
It would be mighty useful to have a button in Desktop View which could close any unwanted desktops. The current function of 'Close empty desktops' from the menu is a somewhat clunky approach.

**Comment by mzomparelli** [comment]
I've been thinking about this myself. Maybe this weekend.

**Comment by JoelAzaria** [comment]
+1 for this.
I noticed too, when I used "close open desktops" that it did something weird with my desktop names when it closed out a desktop in the middle of the order (I think it closed the desktop but left the naming associated with teh desktop numbers so the desktops 'slid' left but names stayed; ie. dt 5 became 4 and took 4's name, dt6 became 5 and now had 5's name, dt6's name just dissappeared )

**Comment by mzomparelli** [comment]
Yep, that's a bug. Let me deal with that and I'll get back to the enhancement part later.
